lambda/executor.js

```diff
@@ -174,6 +166,16 @@ const runInSeries = async({num, lambdaARN, lambdaAlias, payloads, preARN, postAR
     return results;
 };
 
+const handleLambdaInvocationError = (invocationResults, invocationMode, actualPayload, disablePayloadLogs) => {
+    const parsedResults = JSON.parse(Buffer.from(invocationResults.Payload));
+    let errorMessage = `Invocation error (running in ${invocationMode})`;
+    if (!disablePayloadLogs) {
+        errorMessage += ` with payload ${JSON.stringify(actualPayload)}`;
+    }
+    errorMessage += ` - original error type: "${parsedResults.errorType}", original error message: "${parsedResults.errorMessage}", trace: "${parsedResults.stackTrace}"`;
```

Comment:
Alternative would be to just pass the response Payload as is
`errorMessage +=  parsedResults;`

Reply:
If you just do `errorMessage +=  parsedResults;`, you end up with `[object Object]`, so we'd need to JSON.stringify it back into a string.

Let's keep your current implementation, which is more explicit 👍